src/xcode/ENA/ENA/Source/Models/Metadata/KeySubmissionMetadataService.swift

```diff
@@ -0,0 +1,88 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class KeySubmissionService {
+	private var secureStore: Store
```

Comment:
Please Add Unit testing for all the internal functions inside the service, you can user a MockStore to check if the values are saved correctly

Reply:
Created a separate Task for Unit Testing.